doc/docs/master/how-tos/mount-volume.md

```diff
@@ -0,0 +1,159 @@
+
+   For testing purposes, you might want to add an `index.html`
+   file as described in [Create an index.html file](https://kubernetes.io/docs/tasks/configure-pod-container/configure-persistent-volume-storage/#create-an-index-html-file-on-your-node).
+
+1. Get the ReplicaController (RC) manifest from your pipeline:
+
+   ```bash
+   kubectl get rc <rc-pipeline> -o json > <filename>.yaml
+   ```
+
+   **Example:**
+
+   ```bash
+   kubectl get rc pipeline-edges-v7 -o json > test-rc.yaml
+   ```
+
+1. Open the generated RC manifest for editing.
+1. Under `spec`, find the `volumeMounts` section.
+1. Add your volume in the list of mounts. 
+
+   **Example:**
+
+   ```bash
+   {
+        "mountPath": "/data",
+        "name": "task-pv-storage"
+   }
+   ```
+
+   `mountPath` is where your volume will be mounted inside of the
+   container.
+
+1. Find the `volumes` section:
+1. Add the information about the volume.
+
+   **Example:**
+
+   ```bash
+   {
+        "name": "task-pv-storage",
+        "persistentVolumeClaim": {
+            "claimName": "task-pv-claim"
+        }
+   }
+   ```
+
+   In this section, you need to specify the PersistentVolumeClaim you have
+   created in Step 1.
+
+1. Save these changes to a new file.
+1. Copy the contents of the original RC to clipboard:
+
+   **Example:**
+
+   ```bash
+   pbcopy < test.yaml
+   ```
+
+1. Go to a JSON patch generator, such as [JSON Patch Generator](https://extendsclass.com/json-patch.html),
+and paste the contents of the original RC manifest to the **Source JSON**
+field.
+1. Copy the contents of the modified RC manifest to clipboard
+by using `pbcopy` as described above.
+1. Paste the contents of the modified RC manifest to the **Target JSON**
+field.
+1. Copy the generated JSON Patch.
+1. Go to your terminal and open the pipeline manifest for editing.
+
+   For example, if you are modifying the `edges` pipeline, open the
+   `edges.json` file.
+
+1. Add the patch as a one-liner under the `pod_patch` parameter.
+
+   **Example:**
+
+   ```bash
+   "pod_patch": "[{\"op\": \"add\",\"path\": \"/volumes/-\",\"value\": {\"name\": \"task-pv-storage\",\"persistentVolumeClaim\": {\"claimName\": \"task-pv-claim\"}}}, {\"op\": \"add\",\"path\": \"/containers/0/volumeMounts/-\",\"value\": {\"mountPath\": \"/data\",\"name\": \"task-pv-storage\"}}]"
+   ```
+
+   You need to add a backslash (\) before every quote (") sign
+   that is enclosed in square brackets ([]). Also, you might need
+   to modify the path to `volumeMounts` and `volumes` by removing
+   the `/spec/template/spec/` prefix and replacing the assigned
+   volume number with a dash (-). For example, if a
+   path in the JSON patch is `/spec/template/spec/volumes/5`, you
+   might need to replace it with `/volumes/-`. See the example
```

Comment:
When would you want to do this vs not do this?

Reply:
I think we resolved it.